cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
+func delayPrint(message string, delay time.Duration) chan bool {
```

Comment:
No the benefit to UI was indicated by @krishnasrinivas .  So it was incorporated . Let me ask him to chime in here.